src/completions.jl

```diff
@@ -1,6 +1,25 @@
-### baseline completions ###
+import REPL.REPLCompletions, FuzzyCompletions
 
-handle("completions") do data
```

Comment:
Imho we should keep the current handler name for compatibility with older julia-client versions. Maybe have `is_fuzzy` or whatever in `data` instead of two different handlers?

Reply:
I just prioritized my preference for namings basically 😛. Will change the handler names so that we can preserve the compatibility.